core/src/test/java/org/elasticsearch/search/aggregations/EquivalenceIT.java

```diff
@@ -102,7 +114,9 @@ public void testRandomRanges() throws Exception {
             source = source.endArray().endObject();
             client().prepareIndex("idx", "type").setSource(source).execute().actionGet();
         }
-        assertNoFailures(client().admin().indices().prepareRefresh("idx").setIndicesOptions(IndicesOptions.lenientExpandOpen()).execute().get());
+        assertNoFailures(client().admin().indices().prepareRefresh("idx").
+                setIndicesOptions(IndicesOptions.lenientExpandOpen())
+                .execute().get());
```

Comment:
Can you `s/.execute().get()/.get()/`? I try to do that if I'm going to be touching the line.